### PR TITLE
add flavor priority.

### PR DESCRIPTION
**Pull request by Alice1319**
fixes eclipse-xpanse/xpanse#1626

scale:
image

order:
image

migrate:
https://github.com/eclipse-xpanse/xpanse-ui/assets/42802200/7e3ddcd9-ac86-476a-a31b-af4b95a1fe8d

updated:
catalog:

review:

registered service:

register:

### CODE REVIEW

File: src/components/content/common/ocl/FlavorsText.tsx [hunk 1]
```diff
@@ -13,23 +13,30 @@ export function FlavorsText({ flavors }: { flavors: ServiceFlavor[] }): React.JS
...
                         content={
                             <pre className={oclDisplayStyles.oclFlavorsItemContent}>{yamlDocument.toString()}</pre>
                         }
                         title='Flavor Details'
                         trigger='hover'
                     >
-                        <Button className={oclDisplayStyles.oclDataHover} type='link'>
-                            {flavor.name}
-                        </Button>
+                        <Badge
```
Comment (swaroopar) [inline src/components/content/common/ocl/FlavorsText.tsx:31]: this is broken on small screens. The lines are extending outside the table and even the flavor names are outside the column space. 

If this not possible to fit flavors in the same table, you can consider adding flavor as a separate section below the `basic information`. and display flavors similar to new order flavor selection. (Except we do not need the prices inside it)

image

**Review by swaroopar — CHANGES_REQUESTED** [review]

File: src/components/content/order/common/FlavorSelection.tsx [hunk 2]
```diff
@@ -48,6 +50,9 @@ export const FlavorSelection = ({
                                     if (onChangeFlavor) {
```
[the remark this answers is not in the document]
Comment (Alice1319) [reply src/components/content/order/common/FlavorSelection.tsx:50]: fixed

File: src/components/content/common/ocl/FlavorsText.tsx (hunk 1 above)
Comment (Alice1319) [reply src/components/content/common/ocl/FlavorsText.tsx]: register:
image

catalog:
image

review service details:
image

registered service:
image

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**